### ISSUE TITLE
Invalid object name 'information_schema.schemata'.

### ISSUE THREAD

**Issue by 0x4e3** [tds-fdw/tds_fdw#146]
Hi!
I'm trying to import foreign schema from MS SQL 2008 R2, but import fails with following error:
```
NOTICE:  DB-Library notice: Msg #: 208, Msg state: 1, Msg: Invalid object name 'information_schema.schemata'., Server: MSSQL, Process: , Line: 1, Level: 16
```
There is no any errors on MSSQL server side.

OS: Mac OS X 10.13.1
tds_fdw version: 2.0.0-alpha.1
freetds version: 1.00.70

After some investigations, I found, that "information_schema.schemata" view in MSSQL master db has name in uppercase. Can this be a reason?

**Comment by elbrusnt** [comment]
- src/tds_fdw.c
**1.  Line 2908** 
change from 

appendStringInfoString(&buf, "SELECT schema_name FROM information_schema.schemata WHERE schema_name = ");

to

appendStringInfoString(&buf, "SELECT schema_name FROM INFORMATION_SCHEMA.SCHEMATA WHERE schema_name = ");

**2. Line 2943 and 2944**
change from 
"FROM information_schema.tables t "
"  LEFT JOIN information_schema.columns c ON "
to
"FROM INFORMATION_SCHEMA.TABLES t "
"  LEFT JOIN INFORMATION_SCHEMA.COLUMNS c ON "


**3.  Rebuild again**

This  is work for me.

**Comment by 0x4e3** [comment]
Thanks, @elbrusnt! It looks like a good workaround, but I hope this or any other fix of this issue will be implemented by the maintainers.
Now I'm looking for some ways to optimise my postgresql <—> mssql workflows, this one looks pretty good, but I'm not a C programmer and, friendly speaking, I don't like to dig in the unknown code.

Lets wait for the maintainers reaction.
And one more time, thank you for work and time!

**Comment by juliogonzalez** [comment]
@0x4e3, can you please share the query you are using to import the schema?

It also would help if you can tell us the value of 'collation_name' for your database from the output of the following query:

```SQL
select name, collation_name from sys.databases
```
If it doesn't work just try:

```SQL
select * from SYS.DATABASES
```
Please also send us the output of:

```SQL
SELECT SERVERPROPERTY('Collation')
```
As far as I remember, standard SQL identifiers are NOT case sensitive unless you enclose them with quotes. At least this is how PostgreSQL works.

For MSSQL I have less experience, but as far as I can see [here](https://stackoverflow.com/questions/3192512/how-to-remove-case-sensitive-check-in-sql-server-2008) it is possible that your setup is enforcing MSSQL to be case sensitive.

As per [this post](https://blog.sqlauthority.com/2008/12/03/sql-server-default-collation-of-sql-server-2008/), MSSQL 2008 should offer CASE INSENSITIVE collation by default.

Let's see if this is your case, and if so, I think @elbrusnt can create a Pull Request (or I can do it if he wants) so @GeoffMontee can approve it.

**Comment by 0x4e3** [comment]
@juliogonzalez thank you for your answer!
I'm not 100% sure about the query, it was few weeks ago and there's no history, but it was something like this 
```sql
IMPORT FOREIGN SCHEMA msschema01 FROM SERVER sql01
    INTO pgschema01
    OPTIONS (import_default 'true');
```
about collations: server collation is ```Cyrillic_General_CI_AS```, but target source database has ```Cyrilic_General_CS_AS``` -- it's application specific collations, and I'm unable to do anything with it.

**Comment by juliogonzalez** [comment]
@0x4e3, I will check a couple of instances later and will create a PR with @elbrusnt suggestion.

Since 2.0.0 is alpha, I would suggest @GeoffMontee to create a minor version for 1.x after PR is merged since in the end is bugfixing (or I can do it if he's happy about it) and it will make user's life much easier.

**Comment by juliogonzalez** [comment]
1.x code is not affected by this problem (only tests, fixed at #151).

2.0.0 is fixed at #150, but since we should have a way for the people to migrate, I suggest we replace 2.0.0-alpha1 with 2.0.0-alpha2. @GeoffMontee, I can do it after PR is merged unless you have anything against.

**Comment by juliogonzalez** [comment]
@elbrusnt, @0x4e3, 2.0.0-alpha2 is now released, and so I am closing the issue as solved.

Check this version with. If anything is still not working feel free to reopen.
